<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressWindow</class>
 <widget class="QWidget" name="ProgressWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sorting Progress</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="progressLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>8</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>8</height>
        </size>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    border: none;
    background: #dcdcdc;
    border-radius: 4px;
}
QProgressBar::chunk {
    background-color: #0078d7;
    border-radius: 4px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPercentage">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statsLayout">
     <item>
      <widget class="QLabel" name="labelRemaining">
       <property name="text">
        <string>üîÑÔ∏è Remaining: -</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSorted">
       <property name="text">
        <string>   ‚úÖ Sorted: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFailed">
       <property name="text">
        <string>   ‚ùå Failed: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTime">
       <property name="text">
        <string>‚åõTime remaining: -</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLogs">
     <property name="title">
      <string>üìù Log Output</string>
     </property>
     <layout class="QVBoxLayout" name="logLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditLogs">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
        <property name="overwriteMode">
         <bool>false</bool>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>groupBoxLogs</zorder>
  <zorder>progressBar</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
